stable/nginx-ingress/README.md

```diff
@@ -64,6 +64,7 @@ Parameter | Description | Default
 `controller.service.externalIPs` | controller service external IP addresses | `[]`
 `controller.service.loadBalancerIP` | IP address to assign to load balancer (if supported) | `""`
 `controller.service.loadBalancerSourceRanges` | list of IP CIDRs allowed access to load balancer (if supported) | `[]`
+`controller.service.loadBalancerTerminateSSL` | terminates SSL at load balancer (forwards 443 to 80) | false
```

Comment:
I am definitely open to renaming the option (maybe `loadBalancerForwardAllToHTTP`??). I believe you are correct that by default SSL terminates at the ingress controller. The ingress controller by default accesses the endpoints via HTTP (unless configured otherwise: [secure-backends](https://github.com/kubernetes/ingress/blob/master/controllers/nginx/configuration.md#secure-backends)). The purpose of this PR is to allow for the load balancer fronting the ingress controller to be accessed via HTTPS, but forward the traffic to the ingress controller's HTTP port. This would effectively move the SSL termination up the stack from the ingress controller to the load balancer.